src/docs/asciidoc/deployment/configuration/configuration.adoc

```diff
@@ -407,6 +407,7 @@ The card external diffusion service can be configured with the following propert
 |operatorfabric.cardsExternalDiffusion.defaultConfig.activateCardsDiffusionRateLimiter|true|no|Flag to activate mail sending rate limiting
 |operatorfabric.cardsExternalDiffusion.defaultConfig.sendRateLimit|100|no|Max number of mail sent allowed for a single destination in the configured period
 |operatorfabric.cardsExternalDiffusion.defaultConfig.sendRateLimitPeriodInSec|3600|no|Time period for rate limiting control
+|operatorfabric.cardsExternalDiffusion.defaultConfig.defaultTimeZone|Europe/Paris|no|Timezone used to display dates and times for email notifications
```

Comment:
Default timezone used to display dates and times for email notifications if user has not set the timezone for mail  in his settings

Reply:
Done.